src/main/app/src/app/informatie-objecten/informatie-object-create-attended/informatie-object-create-attended.component.ts

```diff
@@ -128,7 +128,7 @@ export class InformatieObjectCreateAttendedComponent
       .build();
 
     const beginRegistratie = new DateFormFieldBuilder(moment())
-      .id("creatiedatum")
+      .id("creationDate")
```

Comment:
I was wondering: instead of using text labels here can we not increase type safety by using `DocumentCreationData.creationDate` here? Or speaking of that: why do we even have this `DocumentCreationData` class because we generate all types from the backend OpenAPI spec in `zac-openapi-types.d.ts`? I see this all over the place.

Reply:
What do you think @infonl-marcel @xiduzo ? for another PR anyway I would say

Reply:
`we not increase type safety by using DocumentCreationData.creationDate here`; we'll have to look into this.

`why do we even have this DocumentCreationData class`; we only started to introduce  using the `zac-openapi-types.d.ts` recently in the frontend, so there are tons of placed which can be enhance by generated typing.

Reply:
ok, clear! ps: Rick introduced the `zac-openapi-types.d.ts` and related proces quite some time ago (6 months or so?). So I kinda would have expected that we would use it for this new SmartDocuments functionality. :-)

Reply:
Herstel: we introduced `zac-openapi-types.d.ts` well over a year ago. :-)